pkg/controller/middleware/method.go

```diff
@@ -19,7 +19,7 @@ import (
 	"net/http"
 	"strings"
 
-	"github.com/google/exposure-notifications-verification-server/pkg/logging"
+	"github.com/google/exposure-notifications-server/pkg/logging"
```

Comment:
nit: consistent whitespace missing.

Reply:
It's actually consistent now 😄 . It goes:

- Group 0: stdlib package
- Group 1: internal packages
- Group 2: main server packages
- Group 3: external packages

We were pretty inconsistent about it, so I fixed it in all the files that this touches.

Reply:
yeah, there's no whitespace between group 2 and 3 in this file, while there is in all other files in this commit.